Sources/ArcGISToolkit/Components/FeatureFormView/FormViewModel.swift

```diff
@@ -42,9 +42,6 @@ import SwiftUI
     /// The list of expression evaluation errors.
     @Published var expressionEvaluationErrors = [FormExpressionEvaluationError]()
     
-    /// A Boolean value indicating whether evaluation is running.
-    @Published var isEvaluating = true
```

Comment:
This was originally going to be used for displaying a progress indicator for long-running expressions. However, it's currently not being used, so I'm OK with removing it.